scripts/Firewall-Automation/Firewall-Automation-Infrastructure-Variables-Template.txt

```diff
@@ -0,0 +1,12 @@
+
+export ASB_FW_Tenant_Id='<<TENANTID>>'
+export ASB_FW_Subscription_Name='<<SUBSCRIPTION-NAME>>'
+export ASB_FW_Deployment_Name='<<FW_DEPLOYMENT_NAME>>'
+export ASB_FW_Base_Automation_System_Name='firewall-automation'
+export ASB_FW_Environment='<<FW_ENV>>'
+export ASB_FW_PowerShell_Runbook_File_Name='Firewall-Automation-Runbook.ps1'
+export ASB_FW_Sku='Basic'
+export ASB_FW_Location='<<FW_LOCATION>>'
+export ASB_FW_PowerShell_Runbook_Description='This runbook automates the allocation and de-allocation of a firewall for the purposes of scheduling.'
+export ASB_SP_CONNECT_AZ_CLIENTID='<<SP-CLIENT>>'
+export ASB_SP_CONNECT_AZ_SECRET='<<SP-SECRET>>'
```

Comment:
Non-blocking recommendation for future consideration.

I recommend adding variables here for resources that are expected to already exist like the firewall, instead of generating the expected names in the different scripts. One example is the firewall variables in `scripts/Firewall-Automation/Firewall-Automation-Runbook.ps1`

Instead of doing value generation in the script like this,

```powershell
$firewallResourceGroup = "rg-" + $baseName + "-" + $environment + "-hub"
$firewallName = "fw-" + $location
```
do it instead in the env file or before.
```bash
export ASB_FW_RESOURCE_GROUP="rg-${baseName}-${environment}-hub"
export ASB_FW_NAME="fw-${location}"
```
This would allow the setup to be more flexible when resources names don't exactly line up with our existing naming conventions.

Reply:
Great suggestion, thank you.